api/tests/acceptance/application/user-orga-settings-controller_test.js

```diff
@@ -188,4 +188,77 @@ describe('Acceptance | Controller | user-orga-settings-controller', () => {
       });
     });
   });
+
+  describe('PUT /api/user-orga-settings/{id}', () => {
+
+    let expectedOrganizationId;
+
+    beforeEach(async () => {
+      userId = databaseBuilder.factory.buildUser().id;
+      const actualOrganizationId = databaseBuilder.factory.buildOrganization().id;
+      expectedOrganizationId = databaseBuilder.factory.buildOrganization().id;
+      databaseBuilder.factory.buildMembership({ userId, organizationId: actualOrganizationId, organizationRole: 'MEMBER' });
+      databaseBuilder.factory.buildMembership({ userId, organizationId: expectedOrganizationId, organizationRole: 'MEMBER' });
+      databaseBuilder.factory.buildUserOrgaSettings({ userId, currentOrganizationId: actualOrganizationId });
+      await databaseBuilder.commit();
+
+      options = {
+        method: 'PUT',
+        url: `/api/user-orga-settings/${userId}`,
+        headers: { authorization: generateValidRequestAuthorizationHeader(userId) },
+        payload: {
+          data: {
+            relationships: {
+              organization: {
+                data: {
+                  id: expectedOrganizationId,
+                  type: 'organizations'
+                }
+              }
+            }
+          }
+        }
+      };
+    });
+
+    describe('Resource access management', () => {
```

Comment:
Le nom des modules "describe" me paraît un peu vague, on a souvent des "when" dans les describe d'habitude. Ex : 
```javascript
describe('When user is not authenticated', () => {
       it('should respond with a 401 status code (unauthorized access)', async () => {
              // ....
       });
});
describe('When user is authenticated', () => {
       it('should respond with a 200 status code', async () => {
              // ....
       });
});
describe('When the user is not member of organization', () => {
       it('should respond with a 422 status code', async () => {
              // ....
       });
});

Reply:
ma règle perso, c'est que quand il n'y a que 1 `it` dans un contexte, je ne m'embête pas avec un `context` et je mets direct le when à la fin du `it`.

Reply:
J'ai modifié, en mettant les when à la fin du `it`.